Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -24,13 +24,80 @@
 #if ENABLE_SWIFTUI
 
 import Foundation
+import Observation
+import SwiftUI // FIXME: Do not import SwiftUI in WebKit proper.
 
 @_spi(Internal)
```

Comment:
I thought we were also going to just stick with `@_spi(Private)` for now?

Reply:
oops, I keep mixing up Internal and Private